.github/workflows/release-pipeline.yml

```
@@ -11,11 +11,29 @@ concurrency:
   cancel-in-progress: false  
 
 jobs:
+  # Check if pipeline should be skipped based on first line of commit message
+  check-skip:
+    runs-on: ubuntu-latest
+    outputs:
+      should-skip: ${{ steps.check.outputs.should-skip }}
+    steps:
+      - name: Check if pipeline should be skipped
+        id: check
+        run: |
```

actionlint (1.7.7)

"github.event.head_commit.message" is potentially untrusted. avoid using it directly in inline scripts. instead, pass it through an environment variable. see https://docs.github.com/en/actions/security-for-github-actions/security-guides/security-hardening-for-github-actions for more details

(expression)